src/tcp.c

```diff
@@ -241,7 +241,17 @@ static int tcp_fsm(struct tcp_conn_tcb *conn, struct tcp_hdr *rs)
         return tcp_hdr_size(rs);
     case TCP_ESTABLISHED:
         LOG(LOG_INFO, "TCP state: TCP_ESTABLISHED");
-
+        if (rs->tcp_flags & TCP_ACK && rs->tcp_flags & TCP_PSH &&
```

Comment:
Use parentheses `(` `)` to emphasize each condition in this statement.

Reply:
Ok, will fix it in the later commit.